Source/Scene/MetadataSemantic.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * An enum of built-in semantics.
+ *
+ * @exports MetadataSemantic
+ *
+ * @private
+ */
+var MetadataSemantic = {
```

Comment:
Yeah, we'll need to revise this list once more built-in semantics are added.

The point cloud ones do not need to be added because those will end up being glTF vertex attributes rather than metadata per-se.